### PR TITLE
Atlantis Loans && BobaDAO && NileRiver TVL Adapters

### PR DESCRIPTION
**Pull request by Mohammed18062020**
##### Twitter Link:
Atlantis Loans -> https://twitter.com/atlantis_loans
BobaDAO -> https://twitter.com/bobanetwork
NileRiver Finance -> https://twitter.com/NileRiver_Labs

##### Website Link:
Atlantis Loans -> https://atlantis.loans/app
BobaDAO -> https://boba.network/
NileRiver Finance -> https://www.nileriver.finance/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
Atlantis Loans -> 
image

BobaDAO -> 
image

NileRiver Finance -> 
image

##### Current TVL:
Atlantis Loans -> $183,184,098.08
BobaDAO -> $44.15M
NileRiver Finance -> $6,176,775

##### Chain:
Atlantis Loans -> BSC
BobaDAO -> Ethereum
NileRiver Finance -> Moonriver

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
Atlantis Loans -> "id": "atlantis-loans"
BobaDAO -> 
NileRiver Finance -> 

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
Atlantis Loans -> "id":13121
BobaDAO -> 
NileRiver Finance -> 

##### Token address and ticker if any:
Atlantis Loans Token (ATL) -> 0x1fD991fb6c3102873ba68a4e6e6a87B3a5c10271
BobaDAO Token (BOBA) -> 
NileRiver Finance (NIL) -> 0xc64B360913BA4500E93E9b68dBf8dD20BDb57Dc0

##### Category (Yield/Dexes/Lending/Minting/Assets/Insurance/Options/Indexes/Staking) *Please choose only one:
Atlantis Loans -> Lending
BobaDAO -> Dexes
NileRiver Finance -> Yield

### CODE REVIEW

**Comment by 0xngmi** [comment]
> nile river

i think they are double counting tvl, once on the liquidity and then again in the farms page

> boba

Seems like this is counting tokens on the bridge, we are not doing this for other protocols so i think it would be best to not count the bridge for boba either

File: projects/atlantisloans/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "bsc",
+    transformAddress,
+    "0x5A9A90983A369b6bB8F062f0AFe6219Ac01caF63",
+    "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c"
+  )(...params);
+}
+
+module.exports = {
+  misrepresentedTokens: true,
+  staking: {
```
Comment (0xngmi) [inline projects/atlantisloans/index.js:28]: would be better to put all these extra exports under the bsc object

**Comment by Mohammed18062020** [comment]
> > nile river
> 
> i think they are double counting tvl, once on the liquidity and then again in the farms page
> 
> > boba
> 
> Seems like this is counting tokens on the bridge, we are not doing this for other protocols so i think it would be best to not count the bridge for boba either

NileRiver: You are rigth I thougth that as well, but as I saw the difference between the tvl total on pools and the tvl total on farms... at anycase, I'm going to leave just the farms, it seems more logical.

 Boba: this protocol is in process, then all that quantity just came from user staking who want the boba token when it's lunched. Then, I count that for this moment and then we're going to update the protocol.

**Comment by 0xngmi** [comment]
yeah i have no clue how farms tvl is higher than liquidity, given that all farms use tokens that are part of liquidity, so it should be a subset

**Pull request merged**